### PR TITLE
clarified settle timeout check

### PR DESCRIPTION
**Pull request by hackaugusto**
Tried to clarify the comments about the settlement timeout check.

### CODE REVIEW

File: raiden/channel/netting_channel.py [hunk 1]
```diff
@@ -524,9 +524,15 @@ def register_transfer_from_to(
 
                 raise InvalidLocksRoot(expected_locksroot, transfer.locksroot)
 
-            # For the sender: If the lock expires after the settle_period a
-            # secret could be revealed after the channel is settled and we
-            # won't be able to claim the token.
+            # For the mediators: This is registering the the *mediator* paying
```
Comment (czepluch) [inline raiden/channel/netting_channel.py]: the the *mediator*. Remove one `the`

File: raiden/channel/netting_channel.py [hunk 2]
```diff
@@ -524,9 +524,15 @@ def register_transfer_from_to(
 
                 raise InvalidLocksRoot(expected_locksroot, transfer.locksroot)
 
-            # For the sender: If the lock expires after the settle_period a
-            # secret could be revealed after the channel is settled and we
-            # won't be able to claim the token.
+            # For mediators: This is registering the the *mediator* paying
```
Comment (LefterisJP) [inline raiden/channel/netting_channel.py]: `the the *mediator* paying` -> `the *mediator* paying`

File: raiden/channel/netting_channel.py [hunk 3]
```diff
@@ -524,9 +524,15 @@ def register_transfer_from_to(
...
+            # transfer, the expiration of the lock must be `reveal_timeout`
```
Comment (LefterisJP) [inline raiden/channel/netting_channel.py]: `transfer, the ` -> `transfer. The`

File: raiden/channel/netting_channel.py [hunk 4]
```diff
@@ -524,9 +524,15 @@ def register_transfer_from_to(
...
+            # blocks smaller than the *received* paying transfer, this cannot
```
Comment (LefterisJP) [inline raiden/channel/netting_channel.py]: ` transfer, this` ->  `transfer. This`

File: raiden/channel/netting_channel.py [hunk 5]
```diff
@@ -524,9 +524,15 @@ def register_transfer_from_to(
...
+            # be checked by the paying channel alone.
+            #
+            # For the initiators: There is no backing transfer, the expiration
```
Comment (LefterisJP) [inline raiden/channel/netting_channel.py]: Can you please rephrase to: 
`For the initiators: As there is no backing transfer, the expiraton is arbitrary. Using the ...`

File: raiden/tests/integration/test_transfers.py [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+from raiden.tests.utils.network import CHAIN
+
+
+@pytest.mark.parametrize('privatekey_seed', ['event_new_channel:{}'])
+@pytest.mark.parametrize('number_of_nodes', [4])
+@pytest.mark.parametrize('channels_per_node', [CHAIN])
+@pytest.mark.parametrize('settle_timeout', [50])
+@pytest.mark.parametrize('reveal_timeout', [5])
+def test_identifier_collision(raiden_chain, token_addresses, deposit):
+    """ Regression test for transfer identifier collision. """
```
Comment (LefterisJP) [inline raiden/tests/integration/test_transfers.py]: Perhaps add a small comment here: That identifier collision should have no adverse effect in mediated transfers and this is what you test here?

File: raiden/tests/integration/test_transfers.py [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+from raiden.tests.utils.transfer import channel
+from raiden.tests.utils.network import CHAIN
+
+
+@pytest.mark.parametrize('privatekey_seed', ['event_new_channel:{}'])
+@pytest.mark.parametrize('number_of_nodes', [4])
+@pytest.mark.parametrize('channels_per_node', [CHAIN])
+@pytest.mark.parametrize('settle_timeout', [50])
+@pytest.mark.parametrize('reveal_timeout', [5])
+def test_identifier_collision(raiden_chain, token_addresses, deposit):
```
Comment (LefterisJP) [inline raiden/tests/integration/test_transfers.py]: Appreciate the test! But it has nothing to do with the PR, which was to clarify some comments only. Could have easily been its own PR and its own commit.

Comment (hackaugusto) [inline raiden/tests/integration/test_transfers.py]: Oops, ignore this file, It was a mistake from my part as the test is not done

**Pull request merged**